edge_server/chronos/lib/config.py

```diff
@@ -54,5 +55,11 @@ def _wrap(self, value):
         else:
             return Struct(value) if isinstance(value, dict) else value
 
-
-cfg = Struct(config_dict)
+import random
```

Comment:
I've just put this here for now to drop the error text in a development deployment with the dockerized edge_server